server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -152,74 +160,136 @@ public void setupEach() throws IOException {
     userController = new UserController(db);
   }
 
-  @Test
-  public void getAllUsers() throws IOException {
+  /**
+   * Construct an instance of `ContextUtil`, which is essentially
```

Comment:
This isn't creating an instance of `ContextUtil` but rather an instance of `Context`.

Reply:
Good catch! An example of updating code without updating the associated comments.

Reply:
This same comment is also wrong in Lab 4. `ContextUtil.init` is a static method that returns an instance of `Context`